source/errors.rst

```diff
@@ -272,7 +272,7 @@ is already used by another user. Compare 8. Slug Already Exists.
 
   {
     "status": 409,
-    "error": "Username already exists",
-    "text": "username ${username} already exists",
+    "error": "Username Already Exists",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L222 shows that `'Username already exists'` is the correct form.